go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/apache/skywalking-cli
 go 1.13
 
 require (
+	github.com/99designs/gqlgen v0.11.3 // indirect
```

Comment:
Is this included in the License?

Reply:
This one is only needed in code generation phase, do we need to add it in the license? If so, I'll add it

Reply:
In that case, no. Code generation is only a tool, not dependency. Unless like Java antlr, if runtime is required, then should add into license.